src/config/scrivito.js

```diff
@@ -15,3 +15,9 @@ if (process.env.SCRIVITO_PRERENDER) {
 }
 
 Scrivito.configure(config);
+
+Scrivito.configureObjClassForContentType({
```

Comment:
* [ ] FYI the config file structure is less clear now.

**Before:**
* `Scrivito.configure` lives in `config/scrivito`.
* `Scrivito.configureContentBrowser` lives in `config/scrivitoContentBrowser`.

**Now:**
* `Scrivito.configure` lives in `config/scrivito`,
* `Scrivito.configureContentBrowser` lives in `config/scrivitoContentBrowser`.
* `Scrivito.configureObjClassForContentType` lives in `config/scrivito`.
 
As the reader, I wonder what is the rule here?